diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/KotlinParser.kt

```diff
@@ -0,0 +1,104 @@
+import org.jetbrains.kotlin.com.intellij.pom.tree.TreeAspect
+import org.jetbrains.kotlin.config.CompilerConfiguration
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.psi.KtPsiFactory
+import org.jetbrains.kotlin.resolve.ImportPath
+import sun.reflect.ReflectionFactory
+
+class KotlinParser {
+
+    /**
+     * Set idea.io.use.nio2 in system property to true
+     * fixme Maybe should check OS to don't change system property
+     */
+    init {
+        setIdeaIoUseFallback()
+    }
+
+    fun createNode(text: String, isPackage: Boolean = false): ASTNode {
+        return makeNode(text, isPackage) ?: throw KotlinParseException("Your text is not valid")
+    }
+
+    /**
+     * This method create a node based on text.
+     * @param isPackage - flag to check if node will contains package.
+     * If this flag is true, node's element type will be FILE.
+     * Else, try to create node based on text.
+     * If this node will contain ERROR_ELEMENT type children this mean that cannot create node based on this text
+     */
+    private fun makeNode(text: String, isPackage: Boolean = false): ASTNode? {
+        val compilerConfiguration = CompilerConfiguration()
+        compilerConfiguration.put(CLIConfigurationKeys.MESSAGE_COLLECTOR_KEY, MessageCollector.NONE) // mute the output logging to process it themselves
+        val pomModel: PomModel = object : UserDataHolderBase(), PomModel {
+            override fun runTransaction(transaction: PomTransaction) {
+                (transaction as PomTransactionBase).run()
+            }
+            @Suppress("UNCHECKED_CAST")
+            override fun <T : PomModelAspect> getModelAspect(aspect: Class<T>): T? {
+                if (aspect == TreeAspect::class.java) {
+                    val constructor = ReflectionFactory.getReflectionFactory().newConstructorForSerialization(
+                            aspect,
+                            Any::class.java.getDeclaredConstructor(*arrayOfNulls<Class<*>>(0))
+                    )
+                    return constructor.newInstance() as T
+                }
+                return null
+            }
+        } // I don't really understand what's going on here, but thanks to this, you can use this node in the future
+        val project = KotlinCoreEnvironment.createForProduction(
+                Disposable {},
+                compilerConfiguration,
+                EnvironmentConfigFiles.JVM_CONFIG_FILES
+        ).project // create project
+        project as MockProject
+        project.registerService(PomModel::class.java, pomModel)
+        if (text.isEmpty())
+            return null
+        val ktPsiFactory = KtPsiFactory(project, true)
+        if (text.trim().isEmpty())
+            return ktPsiFactory.createWhiteSpace(text).node
+        lateinit var node: ASTNode
+        node = if (isPackage)
+            ktPsiFactory.createFile(text).node
+        else {
```

Comment:
```suggestion
        node = if (isPackage) {
            ktPsiFactory.createFile(text).node
        } else {
```
Please add braces, it's really hard to read complex expressions without them.